js/script.js

```diff
@@ -1,29 +1,65 @@
-function calcSum() { 
-    const a = +document.getElementById('a').value; 
-    const b = +document.getElementById('b').value;
-    const c1 = a + b;
-    document.getElementById('result').innerHTML = 'result ' + c1;   
- } 
- 
- function calcSub() {
-    const a = +document.getElementById('a').value; 
-    const b = +document.getElementById('b').value;
-    const c2 = a - b;
-    document.getElementById('result').innerHTML = 'result ' + c2;   
- }
+let globalNumbers = document.querySelectorAll('.action-bar__numbers'),
```

Comment:
Use `let` only for mutable variables. In all other cases you should use `const`. Read [this article](https://medium.com/javascript-scene/javascript-es6-var-let-or-const-ba58b8dcde75).